feature-libs/my-account/organization/styles/__ux/_list.scss

```diff
@@ -379,3 +386,26 @@ cx-organization-form {
 //   }
 
 // }
+cx-unit-list {
+  @include media-breakpoint-down(md) {
```

Comment:
we need to revise the css as the right table layout seems the VERTICAL layout. The VERTICAL_STACKED layout that we use by default for the list component doesn't make to much sense here.